stac_fastapi_api/fastapi_stac/api/app.py

```diff
@@ -7,21 +7,24 @@
 from stac_pydantic import ItemCollection
 from stac_pydantic.api import ConformanceClasses, LandingPage
 
-from stac_api.api.extensions import FieldsExtension
-from stac_api.api.extensions.extension import ApiExtension
-from stac_api.api.models import (
+from stac_api.clients.base import BaseCoreClient
+from stac_api.config import ApiSettings, inject_settings
+from stac_api.errors import DEFAULT_STATUS_CODES, add_exception_handlers
+from stac_api.models import schemas
+from stac_fastapi.api.extensions import FieldsExtension
+from stac_fastapi.api.extensions.extension import ApiExtension
+from stac_fastapi.api.models import (
```

Comment:
It's intentional to still import from both `stac_api` and `stac_fastapi`?

Reply:
I'm switching out the imports to `stac_fastapi` as I make the submodules.